portal/index.php

```diff
@@ -159,53 +191,74 @@ function validate_new_pass() {
         return pass;
     }
 </script>
+<style>
+    .table > tbody > tr > td {
+        border-top: 0px;
+    }
+</style>
 </head>
 <body class="skin-blue">
 <br><br>
 <div class="container text-center">
     <?php if (isset($_SESSION['password_update']) || isset($_GET['password_update'])) {
         $_SESSION['password_update']=1;
         ?>
-      <div id="wrapper" class="centerwrapper" style="text-align:center;">
-        <h2 class="title"><?php echo xlt('Please Enter a New Password'); ?></h2>
+      <div id="wrapper" class="centerwrapper text-center">
+        <h2 class="title"><?php echo xlt('Please Enter New Credentials'); ?></h2>
         <form action="get_patient_info.php" method="POST" onsubmit="return process_new_pass()" >
-            <table style="width:100%">
+            <input style="display:none" type="text" name="dummyuname"/>
+            <input style="display:none" type="password" name="dummypass"/>
```

Comment:
Can find where above dummyuname and dummypass are being used.

Reply:
believe it or not but this is a trick to help prevent autofill.